python/foglamp/services/core/server.py

```diff
@@ -115,10 +115,10 @@ class Server:
     rest_server_port = 0
     """ FogLAMP REST API port """
 
-    is_rest_server_http_enabled = False
+    is_rest_server_http_enabled = True
```

Comment:
Not needed, as when server starts it calls def `rest_api_config` (L#278) but yes for better readability we can.